src/NServiceBus.Hosting.Windows/GenericHost.cs

```diff
@@ -88,6 +89,11 @@ Task<IStartableEndpoint> PerformConfiguration(Action<EndpointConfiguration> more
             RoleManager.TweakConfigurationBuilder(specifier, configuration);
             profileManager.ActivateProfileHandlers(configuration);
 
+
+            configuration.RegisterComponents(r => r.ConfigureComponent(() => startableAndStoppableBag, DependencyLifecycle.SingleInstance));
```

Comment:
Think its better to move this into the setup method of the `StartableAndStoppableFeature` ?